tests/autobahn/client/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.9"
```

Comment:
I wonder what would it take to integrate this into pytest

Reply:
@anesabml add a link to the new PR here, once you have it. This is useful for interlinking related changes.

Reply:
Here's is the [PR](https://github.com/anesabml/aiohttp/pull/1)

Reply:
Oh, I thought you'd send it against upstream.

Reply:
Here's the new PR against upstream https://github.com/aio-libs/aiohttp/pull/5809